### ISSUE TITLE
Application x has not been registered

### ISSUE THREAD

**Issue by dstollie** [storybooks/storybook#2254]
After running `getstorybook` and `yarn run storybook` im getting the following error:

![screen shot 2017-11-07 at 09 53 10](https://user-images.githubusercontent.com/7110155/32484830-8546a342-c3a1-11e7-87ba-9e927388dad7.jpg)

# Tried so far
- Restarted mac fully so I 100% know there are no packer dev servers running
- Removing the node_modules folder and download the packages using yarn.
- Run the project from xcode instead of the terminal

# Situation
These are  de react versions I use:
```
    "react": "16.0.0-alpha.12",
    "react-native": "0.48.4",
```
Node : 8.5.0

**Comment by danielduan** [comment]
try `react@16` and `react-native@0.49`. I think you also need an `index.js` file instead of the `index.ios.js` and `index.android.js`.

**Comment by dstollie** [comment]
@danielduan thanks for the tip. It now starts the app successfully but for some reason storybook wont start on the simulator. The app is just start as usual when using `react-native start`

The steps I follow:

1. `yarn run storybook -- --reset-cache`
1. go to http://localhost:7007/ (this page load succesfully)
1. `react-native run-ios`

The app does build and the packer does not give errors but the storybook is not loaded but the usual app

**Comment by danielduan** [comment]
If you aren't running the CRNA version, you have to manually comment out your `index.js` or `index.ios/android.js` and paste in this:
```
import Storybook from './storybook';

export default Storybook;
```

Make sure the `App.json` has the same name and displayName as the index from `./storybook` that you are importing.

**Comment by dstollie** [comment]
Solved the issue! I had to make two changes

1. Renaming './storybook/index.ios.js' to './storybook/index.js' and removing './storybook/index.android.js'

1. The appKey of the app itself was 'Logistiek' whereas the appKey in 'storybook.js' was 'logistiek'. By renaming  'Logistiek' to 'logistiek' everything went OK. For people having the same issue the appKey is being set with the first argument of the 'registerComponent' method

```javascript
AppRegistry.registerComponent('logistiek', () => StorybookUI);
```
